eq-author/src/apollo/createApolloErrorLink.js

```diff
@@ -10,19 +10,30 @@ import { signOutUser } from "../redux/auth/actions";
 export const errorHandler = (getStore, error) => {
   const { networkError, graphQLErrors } = error;
   setSentryUser(window.localStorage.getItem("accessToken"));
+
   if (networkError) {
-    getStore().dispatch(apiDownError());
+    const httpStatusCode = networkError.statusCode;
     setSentryTag("networkError");
     sendSentryError(networkError);
-    if (networkError.bodyText === "User does not exist") {
-      getStore().dispatch(signOutUser());
+
+    switch (httpStatusCode) {
+      // 401 - User does not exist
+      case 401:
+        getStore().dispatch(signOutUser());
+        break;
+      // 403 - Unauthorized questionnaire access
+      case 403:
+        return;
```

Comment:
This should probably be a break instead, just in case there is a networkError and a graphqlError.

Reply:
Changed it to a break.